### PR TITLE
Fix command line option bugs with sunsch

### PR DESCRIPTION
**Pull request by berland**
Uncoved while writing tests for lines that was outside code coverage.

Do that more often.

Kudus codecov.io.

### CODE REVIEW

File: tests/test_sunsch.py [hunk 1]
```diff
@@ -27,13 +27,20 @@ def readonly_datadir():
         os.chdir(cwd)
 
 
-def test_main(tmpdir, caplog, mocker):
-    """Test command line sunsch, loading a yaml file"""
-
+@pytest.fixture
+def testdata(tmpdir):
     tmpdir.chdir()
+    cwd = os.getcwd()
```
Comment (eivindsm) [inline tests/test_sunsch.py:33]: This could probably also be performed with pathlib

**Review by eivindsm — COMMENTED** [review]
I see 6  lines using os in test_sunch.py. Are you not able to lift the os requirement entirely?

File: tests/test_sunsch.py (hunk 1 above)
Comment (berland) [reply tests/test_sunsch.py:33]: Fixed!

Comment (berland) [reply tests/test_sunsch.py:33]: > I see 6 lines using os in test_sunch.py. Are you not able to lift the os requirement entirely?

`os.chdir()` has no pathlib-replacement.

**Review by eivindsm — APPROVED** [review]
LGTM

**Pull request merged**